src/autolua.zig

```diff
@@ -172,6 +172,11 @@ pub fn check(L: ?*lua.lua_State, idx: c_int, comptime T: type) T {
             // lua.lua_pop(L, 1);
             // const ptr = lua.lua_touserdata(L, idx);
         },
+        .Optional => |N| {
+            if (lua.lua_type(L, idx)<lua.LUA_TBOOLEAN)
+                return null;
```

Comment:
Rest of the code uses curly brackets, even for single-line conditional code, so I assume it should be like that here too for consistency.

Reply:
yeah i just added commits to fix that error (2, accidentally)